index.html

```diff
@@ -8,60 +8,101 @@
   </head>
   <body>
     <header>
-        <img src="img/salmon.png" alt="salmon" id="left-fish">
+        <img src="img/salmon.png" alt="salmon"/>
         <div class="header-text-wrapper">
           <h1>Pat's Salmon Cookies!</h1>
           <h2>Trust me, you're gonna love 'em!</h2>
         </div><!-- end of header-text-wrapper div -->
-        <img src="img/salmon.png" alt="salmon"id="right-fish">
+        <img src="img/chinook.jpg" alt="salmon" />
     </header>
-    <img src="img/river.jpg" alt="river" id="river">
-      <div id="fish-container">
-        <img src="img/salmon.png" alt="salmon" id="center-fish">
-      </div>
-    <div class="wrapper">
+    <div id="river">
+      <img src="/img/river.png" alt="river">
```

Comment:
The leading `/` here means this image only works if liveserver is running. Get rid of it for now.